nemo/collections/llm/peft/lora.py

```diff
@@ -112,6 +120,7 @@ class LoRA(PEFT):
         dropout_position (Literal['pre', 'post'], optional): Position for applying dropout.
             Can be 'pre' (before the low-rank projection) or 'post' (after). Defaults to 'pre'.
         a2a_experimental (bool): Enables the experimental All-to-All (A2A) communication strategy. Defaults to False.
+        lora_drype (torch.dtype): Parameter data type for LoRA weights. Default None (will use model's dtype).
```

Comment:
why this change is needed?

Reply:
HF supports loading 4bit weights, the adapter will use the model's dtype (in this case 4bit) which is untrainable -> therefore you have to be able to pass the dtype manually.